modules/rosa-cluster-autoscaler-cli-after.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-cli-after_{context}"]
+= Enable autoscaling after cluster creation with the ROSA CLI
+
+You can use the ROSA CLI (`rosa`) to set cluster-wide autoscaling behavior after cluster creation.
+
+.Procedure
+
+. After you have created a cluster, type the following command. At least one parameter must be set in order for cluster autoscaling to be enabled.
```

Comment:
``At least one parameter must be set in order for cluster autoscaling to be enabled.`` that part is not true in this context. we're specifically creating cluster-autoscaler here

Reply:
Does that make the example in line 19 incorrect?

Reply:
I'm going to say yes and remove the <parameter> bit.

Reply:
when providing parameters it will create the autoscaler
when not providing parameters it will enable interactive mode automatically